dev-system/dev/user.clj

```diff
@@ -29,6 +32,26 @@
           (fn [app-map]
             (u/map-values #(assoc-in % [:log :level] level) app-map))))
 
+(defn set-mode!
+  "Set the mode to use when starting/resetting. Valid options are:
+
+  * :in-memory
+  * :external"
+  [new-mode]
```

Comment:
Right, that's what @chris-durbin mentioned -- I have an update for this change that addresses that.